src/frontends/pytorch/src/transforms/quantized_node_replacer.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "quantized_node_replacer.hpp"
+
+#include <memory>
+#include <utility>
+
+#include "openvino/core/rt_info.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "utils.hpp"
+#include "utils_quantize.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace pass {
+
+QuantizedNodeReplacer::QuantizedNodeReplacer() {
+    auto quantized_pt_node = ov::pass::pattern::wrap_type<ov::frontend::pytorch::QuantizedPtNode>();
+
+    ov::matcher_pass_callback callback = [](ov::pass::pattern::Matcher& m) {
+        auto quantized_pt_node = cast_quantized_fw_node(m.get_match_root());
+        if (!quantized_pt_node)
+            return false;
+
+        auto quantized_input = quantized_pt_node->get_input_node_shared_ptr(0);
+        replace_output_update_name(quantized_pt_node, quantized_input);
```

Comment:
```suggestion
        auto quantized_input = quantized_pt_node->get_input_value(0);
        quantized_pt_node->output(0).replace(quantized_input);
```
I don't think we really need to update name, just replace output should be enough.

Reply:
Applied!